### ISSUE TITLE
Generating docs throws TypeError

### ISSUE THREAD

**Issue by mtbkeith** [fivetran/great_expectations#567]
I profiled a basic Django User table and then this happened: 

```

Move the profiled data and build HTML documentation? [Y/n]: Y

Moving files...

Done.

Building documentation...
Traceback (most recent call last):
...
  File "/Users/myuser/.local/share/virtualenvs/pipeline-XeyG96F8/lib/python3.7/site-packages/great_expectations/render/renderer/other_section_renderer.py", line 261, in _get_column_types
    expected_types = set(evr["expectation_config"]["kwargs"]["type_list"])
TypeError: 'NoneType' object is not iterable
(pipeline) bash-3.2$ great_expectations --version
great_expectations, version 0.7.3
```

subsequent to this; 
```
(pipeline) bash-3.2$ great_expectations profile 
Traceback (most recent call last):
  File "/Users/myuser/.local/share/virtualenvs/pipeline-XeyG96F8/lib/python3.7/site-packages/great_expectations/cli/cli.py", line 254, in profile
    profile_datasource(context, datasource_name, max_data_assets=max_data_assets)
UnboundLocalError: local variable 'max_data_assets' referenced before assignment
(pipeline) bash-3.2$ 
```

**Comment by abegong** [comment]
Thanks for flagging this issue.

This was a bug introduced in the 0.7.3 release last week. (See #561 )
It's patched in PR #563, which will be part of the 0.7.4 release, which we are planning to cut in a few hours.

If the issue persists after that, please speak up.

**Comment by mtbkeith** [comment]
Thanks Abe, I guess I only looked at open issues since the release was so new.  
I'm looking forward to working with great_expectations!
